addons/point_of_sale/models/pos_session.py

```diff
@@ -2020,7 +2020,7 @@ def get_total_discount(self):
         amount = 0
         for line in self.env['pos.order.line'].search([('order_id', 'in', self.order_ids.ids), ('discount', '>', 0)]):
             normal_price = line.qty * line.price_unit
-            normal_price = normal_price + (normal_price / 100 * line.tax_ids.amount)
+            normal_price = normal_price + (normal_price / 100 * sum(line.tax_ids.mapped('amount')))
```

Comment:
This calculation of normal_price is wrong. The correct way is to compute the `price_subtotal_incl` of the line considering without discount.

Reply:
I'm making this comment because it's wrong to assume that tax is always "price excluded". There are taxes that are "price included".